github/actions/client.go

```diff
@@ -260,59 +297,14 @@ func (c *Client) CreateRunnerScaleSet(ctx context.Context, runnerScaleSet *Runne
 		req.Header.Set("User-Agent", c.userAgent)
 	}
 
-	httpClient := c.getHTTPClient()
-
-	resp, err := httpClient.Do(req)
-	if err != nil {
-		return nil, err
-	}
-
-	if resp.StatusCode != http.StatusOK {
-		return nil, ParseActionsErrorFromResponse(resp)
-	}
-	var createdRunnerScaleSet *RunnerScaleSet
-	err = unmarshalBody(resp, &createdRunnerScaleSet)
-	if err != nil {
-		return nil, err
-	}
-	return createdRunnerScaleSet, nil
-}
-
-func (c *Client) UpdateRunnerScaleSet(ctx context.Context, runnerScaleSetId int, runnerScaleSet *RunnerScaleSet) (*RunnerScaleSet, error) {
```

Comment:
Do we not using `UpdateRunnerScaleSet` anywhere? 😄

Reply:
Yeah I was very surprised, but the only use of it was in its own test 😄